index.js

```diff
@@ -1,29 +1,30 @@
-export function isStream(stream) {
+export function isStream(stream, {checkOpen = false} = {}) {
 	return stream !== null
 		&& typeof stream === 'object'
+		&& (stream.writable || stream.readable || !checkOpen)
 		&& typeof stream.pipe === 'function';
 }
 
-export function isWritableStream(stream) {
+export function isWritableStream(stream, {checkOpen = true} = {}) {
 	return isStream(stream)
-		&& stream.writable !== false
```

Comment:
`stream.writable` and `stream.readable` are always booleans.

https://github.com/nodejs/node/blob/ec3040f7210988440ab5a31519c8637599f3bb17/lib/internal/streams/readable.js#L1413

https://github.com/nodejs/node/blob/ec3040f7210988440ab5a31519c8637599f3bb17/lib/internal/streams/writable.js#L1001